Makefile

```diff
@@ -9,6 +9,7 @@ GIT_COMMIT=$(shell git rev-parse HEAD)
 GIT_TAG=$(shell if [ -z "`git status --porcelain`" ]; then git describe --exact-match --tags HEAD 2>/dev/null; fi)
 GIT_TREE_STATE=$(shell if [ -z "`git status --porcelain`" ]; then echo "clean" ; else echo "dirty"; fi)
 PACKR_CMD=$(shell if [ "`which packr`" ]; then echo "packr"; else echo "go run vendor/github.com/gobuffalo/packr/packr/main.go"; fi)
+IS_LINUX=$(shell if [ "`uname -s`" = 'Linux' ]; then echo '1'; else echo '0'; fi)
```

Comment:
I think instead of introducing this variable, we should simply remove CGO_ENABLED from the `cli` target entirely and allow go to build natively when it can.